worldengine/hdf5_serialization.py

```diff
@@ -0,0 +1,220 @@
+import h5py
+from worldengine.version import __version__
+from worldengine.biome import biome_name_to_index, biome_index_to_name
+from worldengine.world import World, Step
+import numpy
+
+
+def save_world_to_hdf5(world, filename):
+    f = h5py.File(filename, libver='latest', mode='w')
```

Comment:
I agree we have several refactorings to do, I think that we should focus on that as soon as we release this version